app/tests/v2/system/test_records.py

```diff
@@ -1,5 +1,204 @@
+import json
 from app.tests.v2.base_test import BaseTest
 
 
 class RecordTest(BaseTest):
-    pass
+    def __init__(self):
+        super().__init__()
+        self.token = self.get_token_on_user_login()
+
+    def test_create_new_incident(self):
+        """
+        Test create new incident POST Endpoint.
+        :return: status code 201 (Created)
+        """
+        r = self.create_incident()
+        expected = {"status": 201,
+                    "data": [{
+                        "id": 3,
```

Comment:
I like your implementation on tests @bl4ck4ndbr0wn, One thing though, any reason for hardcoding the 'id'?. My guess is, the id will change every time you create a new instance and that means this test case will only pass when the `id=3`.